.gitignore

```diff
@@ -19,3 +19,6 @@ src/test/data/sandbox/
 
 # MacOS custom attributes files created by Finder
 .DS_Store
+
+# Custom Windows / Shell scripts
+*.bat
```

Comment:
You have issues with these files? Just checking

Reply:
I wrote a trivial batch script to run the Appveyor tests, didn't want my personal scripts to pollute the repo.